Index: Program/Cura/resources/machine_profiles/Series1-2014.ini
===================================================================
--- /dev/null
+++ Program/Cura/resources/machine_profiles/Series1-2014.ini
@@ -0,0 +1,163 @@
+[profile]
+layer_height = .3
+wall_thickness = 3
+retraction_enable = True
+solid_layer_thickness = 2
+fill_density = 50
+nozzle_size = .5
+print_speed = 200
+print_temperature = 210
+print_temperature2 = 0
+print_temperature3 = 0
+print_temperature4 = 0
+print_bed_temperature = 70
+support = None
+platform_adhesion = Brim
+support_dual_extrusion = Both
+wipe_tower = False
+wipe_tower_volume = 15
+ooze_shield = False
+filament_diameter = 1.75
+filament_diameter2 = 0
+filament_diameter3 = 0
+filament_diameter4 = 0
+filament_flow = 100
+retraction_speed = 80
+retraction_amount = 4.5
+retraction_dual_amount = 16.5
+retraction_min_travel = 1.5
+retraction_combing = True
+retraction_minimal_extrusion = 0.02
+bottom_thickness = .2
+object_sink = 0
+overlap_dual = 0.15
+travel_speed = 100
+bottom_layer_speed = 25
+infill_speed = 0
+cool_min_layer_time = 30
+fan_enabled = True
+skirt_line_count = 1
+skirt_gap = 3.0
+skirt_minimal_length = 150.0
+fan_full_height = 0.5
+fan_speed = 100
+fan_speed_max = 100
+cool_min_feedrate = 10
+cool_head_lift = False
+solid_top = True
+solid_bottom = True
+fill_overlap = 15
+support_fill_rate = 15
+support_xy_distance = 0.7
+support_z_distance = 0.15
+spiralize = False
+brim_line_count = 20
+raft_margin = 5
+raft_line_spacing = 1.0
+raft_base_thickness = 0.3
+raft_base_linewidth = 0.7
+raft_interface_thickness = 0.2
+raft_interface_linewidth = 0.2
+fix_horrible_union_all_type_a = True
+fix_horrible_union_all_type_b = False
+fix_horrible_use_open_bits = False
+fix_horrible_extensive_stitching = False
+plugin_config = (lp1
+	.
+object_center_x = -1
+object_center_y = -1
+
+[alterations]
+start.gcode = 
+	;
+	; *** G-code Prefix ***
+	; For cura, it already sets the temp beforehand
+	; so no need to set temperature here
+	G90 ; use absolute coordinates
+	M82 ; absolute extrusion distances
+	G21 ; set units to millimeters
+	M106 S0 ; fan speed 0
+	; M104 S220 ; Set tempterature 190, don't wait
+	M301 P24.72 I0.88 D231.1  ; set pid, looks like new code is correct.
+	G28  ;  home X Y Z
+	G0 Z20 ;  put bed down a little
+	G0 X120 Y115 Z2 F7500 ; go to 120, 115, 2.
+	G92 E0 ; clear E
+	G1 Y1 Z2 F7500 ;
+	; M109 S220  ; set temp 190 and wait
+	G1 F100 E30
+	G1 F100 Z15 E32
+	G1 F100 E31
+	G1 X100 Z0 E30 F7500
+	G1 X90 Z0 E30 F7500
+	G1 X90 Z0.3 E32 F7500
+	G92 E0  ; clear E
+end.gcode = 
+	; *** Cooling Extruder 1 to 0 C ***
+	; Guaranteed same extruder, cooling down
+	; BfB-style
+	;M104 S0
+	; 5D-style
+	M104 S0
+	;
+	; fan off
+	M107
+	; *** G-code Postfix ***
+	;
+	G1 Y230 F10000
+	G92 E0
+	G1 E10
+	M107
+	M104 S0
+	M84 E X Y
+start2.gcode = ;Sliced at: {day} {date} {time}
+	;Basic settings: Layer height: {layer_height} Walls: {wall_thickness} Fill: {fill_density}
+	;Print time: {print_time}
+	;Filament used: {filament_amount}m {filament_weight}g
+	;Filament cost: {filament_cost}
+	;M190 S{print_bed_temperature} ;Uncomment to add your own bed temperature line
+	;M104 S{print_temperature} ;Uncomment to add your own temperature line
+	;M109 T1 S{print_temperature2} ;Uncomment to add your own temperature line
+	;M109 T0 S{print_temperature} ;Uncomment to add your own temperature line
+	G21        ;metric values
+	G90        ;absolute positioning
+	M107       ;start with the fan off
+	G28 X0 Y0  ;move X/Y to min endstops
+	G28 Z0     ;move Z to min endstops
+	G1 Z15.0 F{travel_speed} ;move the platform down 15mm
+	T1                      ;Switch to the 2nd extruder
+	G92 E0                  ;zero the extruded length
+	G1 F200 E10             ;extrude 10mm of feed stock
+	G92 E0                  ;zero the extruded length again
+	G1 F200 E-{retraction_dual_amount}
+	T0                      ;Switch to the first extruder
+	G92 E0                  ;zero the extruded length
+	G1 F200 E10             ;extrude 10mm of feed stock
+	G92 E0                  ;zero the extruded length again
+	G1 F{travel_speed}
+	;Put printing message on LCD screen
+	M117 Printing...
+end2.gcode = ;End GCode
+	M104 T0 S0                     ;extruder heater off
+	M104 T1 S0                     ;extruder heater off
+	M140 S0                     ;heated bed heater off (if you have it)
+	G91                                    ;relative positioning
+	G1 E-1 F300                            ;retract the filament a bit before lifting the nozzle, to release some of the pressure
+	G1 Z+0.5 E-5 X-20 Y-20 F{travel_speed} ;move Z up a bit and retract filament even more
+	G28 X0 Y0                              ;move X/Y to min endstops, so the head is out of the way
+	M84                         ;steppers off
+	G90                         ;absolute positioning
+support_start.gcode = 
+support_end.gcode = 
+cool_start.gcode = 
+cool_end.gcode = 
+replace.csv = 
+switchextruder.gcode = ;Switch between the current extruder and the next extruder, when printing with multiple extruders.
+	G92 E0
+	G1 E-36 F5000
+	G92 E0
+	T{extruder}
+	G1 X{new_x} Y{new_y} Z{new_z} F{travel_speed}
+	G1 E36 F5000
+	G92 E0
+
Index: Program/Cura/util/profile.py
===================================================================
--- Program.orig/Cura/util/profile.py
+++ Program/Cura/util/profile.py
@@ -157,6 +157,7 @@ setting('retraction_enable',       False
 setting('solid_layer_thickness',     0.6, float, 'basic',    _('Fill')).setRange(0).setLabel(_("Bottom/Top thickness (mm)"), _("This controls the thickness of the bottom and top layers, the amount of solid layers put down is calculated by the layer thickness and this value.\nHaving this value a multiple of the layer thickness makes sense. And keep it near your wall thickness to make an evenly strong part."))
 setting('fill_density',               20, float, 'basic',    _('Fill')).setRange(0, 100).setLabel(_("Fill Density (%)"), _("This controls how densely filled the insides of your print will be. For a solid part use 100%, for an empty part use 0%. A value around 20% is usually enough.\nThis won't affect the outside of the print and only adjusts how strong the part becomes."))
 setting('nozzle_size',               0.4, float, 'advanced', _('Machine')).setRange(0.1,10).setLabel(_("Nozzle size (mm)"), _("The nozzle size is very important, this is used to calculate the line width of the infill, and used to calculate the amount of outside wall lines and thickness for the wall thickness you entered in the print settings."))
+setting('max_print_volume_per_second',0.0, float, 'expert', _('Machine')).setRange(0.0,40.0).setLabel(_("Maximum print volume/sec"), _("This speed will limit how fast your print will go.  It's best to measure this by extruding at different speeds and figuring out your own maximum"))
 setting('print_speed',                50, float, 'basic',    _('Speed and Temperature')).setRange(1).setLabel(_("Print speed (mm/s)"), _("Speed at which printing happens. A well adjusted Ultimaker can reach 150mm/s, but for good quality prints you want to print slower. Printing speed depends on a lot of factors. So you will be experimenting with optimal settings for this."))
 setting('print_temperature',         220, int,   'basic',    _('Speed and Temperature')).setRange(0,340).setLabel(_("Printing temperature (C)"), _("Temperature used for printing. Set at 0 to pre-heat yourself.\nFor PLA a value of 210C is usually used.\nFor ABS a value of 230C or higher is required."))
 setting('print_temperature2',          0, int,   'basic',    _('Speed and Temperature')).setRange(0,340).setLabel(_("2nd nozzle temperature (C)"), _("Temperature used for printing. Set at 0 to pre-heat yourself.\nFor PLA a value of 210C is usually used.\nFor ABS a value of 230C or higher is required."))
Index: Program/Cura/util/validators.py
===================================================================
--- Program.orig/Cura/util/validators.py
+++ Program/Cura/util/validators.py
@@ -134,7 +134,10 @@ class printSpeedValidator(object):
 			printVolumePerSecond = printVolumePerMM * printSpeed
 			#Using 10mm3 per second with a 0.4mm nozzle (normal max according to Joergen Geerds)
 			maxPrintVolumePerSecond = 10 / (math.pi*(0.2*0.2)) * (math.pi*(nozzleSize/2*nozzleSize/2))
-			
+			maxPrintVolumePerSecond_setting = profile.getProfileSettingFloat('max_print_volume_per_second')
+                        if (maxPrintVolumePerSecond_setting > 0):
+                                maxPrintVolumePerSecond = maxPrintVolumePerSecond_setting
+                        #return WARNING, "max print voume = %f mm^3/sec" % (maxPrintVolumePerSecond)
 			if printVolumePerSecond > maxPrintVolumePerSecond:
 				return WARNING, 'You are trying to print more then %.1fmm^3 of filament per second. This might cause filament slipping. (You are printing at %0.1fmm^3 per second)' % (maxPrintVolumePerSecond, printVolumePerSecond)
 			
Index: Program/Cura/current_profile.ini
===================================================================
--- Program.orig/Cura/current_profile.ini
+++ Program/Cura/current_profile.ini
@@ -5,7 +5,8 @@ retraction_enable = True
 solid_layer_thickness = 2
 fill_density = 50
 nozzle_size = .5
-print_speed = 200
+max_print_volume_per_second = 8.0
+print_speed = 53
 print_temperature = 210
 print_temperature2 = 0
 print_temperature3 = 0
